packages/extension-collaboration/src/collaboration.ts

```diff
@@ -9,6 +9,8 @@ import {
 } from 'y-prosemirror'
 import { UndoManager } from 'yjs'
 
+type YSyncOpts = Parameters<typeof ySyncPlugin>[1]
```

Comment:
Do you know where the changes on this file come from? Seems like they are unrelated to your changes, right?